.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode10.1
+osx_image: xcode10.2
```

Comment:
We need https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3647 before using Xcode 10.2 image.

Reply:
It has been merged now. ✨